CHANGELOG.md

```diff
@@ -3,8 +3,14 @@
 The dates are in European standard format where date is presented as **YYYY-MM-DD**.
 TombEngine releases are located in this repository (alongside with Tomb Editor): https://github.com/TombEngine/TombEditorReleases
 
+
 ## [Version 1.8.2]
 
+### changes
```

Comment:
Entries in the changelog should be either in "Bug fixes" or "New features" section - I think that in this case, all three entries can go to "Bug fixes" section. Also, in my opinion, it is not necessary to provide such detailed changelog entries as first two - you can combine it to an entry like "Fixed issues with ladder climbing near slanted ledges".